app/index.html

```diff
@@ -180,6 +180,45 @@ <h5>References</h5>
             </div>
           </div>
         </div>
+        <div class="modal fade" tabindex="-1" id='download'>
+          <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title">
+                    Download chemiscope
+                </h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <p>
+                    Chemiscope consists in a set of JavaScript modules to 
+                    visualize atomic structures and map associated data. 
+                    These modules can be assembled into all sorts of interactive
+                    visualization tools, such as that which you can use on these
+                    pages. 
+                </p>
+
+                <p>
+                    If you are a developer you may be interested in downloading
+                    the <a href="https://github.com/cosmo-epfl/chemiscope"> source code</a>,
+                    which is available on GitHub. 
+                </p>
+
+                <p>
+                    If you are a user, you may want to download a 
+                    <a href="chemiscope_standalone.html" download> 
```

Comment:
Regarding the "recursive" issue, we could have some CSS to hide this and a other part of the page (such as the list of examples) which are not relevant in the standalone mode, which would be activate depending on the `isStandalone` variable. Something like

```html
<div class="hide-if-standalone">

</div>

<script>
if (isStandalone) {
    for (const element of document.getElementsByClassName("hide-if-standalone") {
        element.style.display = "none";
    }
}
</script>
```

Reply:
this would be great. you know how to implement it easily?

Reply:
The code above should work. Do you want to add this to this PR or leave it for later?